README.md

```diff
@@ -87,7 +87,7 @@ git commit -m 'Updated to the latest version'
 
 ### Who uses the template?
 
-Going to use the template? Go on! The only thing we ask - let us know at with a 
+Going to use the template? Go on! The only thing we ask - let us know at with a
```

Comment:
this change was certainly by an automatic method, should I revert this and the other cases similar?